src/store/default-state.js

```diff
@@ -0,0 +1,105 @@
+export const defaultRecomendations = [
```

Comment:
I will make you sad, but these are (maybe with different content) already in `default-content-config.json`